docs/content/tutorial/assets/next-steps.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: "Intro to assets: Next steps | Dagster Docs"
+description: What if you want to do more after completing the asset tutorial?
+---
+
+# Intro to assets: Next steps
+
+🎉 Congratulations! Having reached this far, you now have a working, tested set of software-defined assets.
+
+What if you want to do more?
+
+- **Automating asset materialization** - This tutorial covered how to manually materialize assets. Dagster can also kick off materializations automatically: on fixed [schedules](/concepts/partitions-schedules-sensors/schedules) or when your [sensor](/concepts/partitions-schedules-sensors/sensors) says to.
```

Comment:
🌱

More of just a reminder to myself to mention this in the rewrite, so pls don't action on this: declarative scheduling deserves a mention in the equivalent statement of the new version.

Reply:
I'll add this to the Notion task related to this part of the tutorial so you can keep track of it!